src/Teams.Business/Services/DiagnosticService.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Teams.Data;
+
+namespace Teams.Business.Services
+{
+    public class DiagnosticService : IDiagnosticService
+    {
+        private readonly string pathToConfig;
+        private readonly ApplicationDbContext _dbContext;
+        public DiagnosticService(ApplicationDbContext dbContext)
+        {
+            _dbContext = dbContext;
+            pathToConfig = "appsettings.json";     
+        }
+
+        public async Task<bool> CheckDbConnection()
+        {
+            try
+            {
+                if (await _dbContext.Database.CanConnectAsync())
+                    return true;
+                else
+                    return false;
+            }
+            catch
+            {
+                return false;
+            }
+        }
+
+        public string GetCurrentVersion()
```

Comment:
Please read how to correctly get fields from appsettings and where you should read this file.

Reply:
Okay thank you for your notice. I will read.